collectd-docker/docs/cpu.throttling_data.periods.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Periods
+brief: Number of periods
+metric_type: cumulative_counter
+---
+### Periods
+
+(Optional metric) Tracks the cumulative count of number of periods. Docker reports 0 if there is no quota constraint on the container.
```

Comment:
Maybe change the first sentence in these metric decriptions to say "A cumulative count of the number of periods".  @molner is a better wordsmith than I am.

Reply:
I agree with @charliesignalfx  - get rid of the first word of "tracks" and use something along the lines of what he suggested.